packages/oidc/src/dcr/clientRegistrar.spec.ts

```diff
@@ -93,6 +94,28 @@ describe("registerClient", () => {
   // TODO: this only tests the minimal registration request.
   // The additional provided options will be tested in an upcoming PR.
 
+  it("does not send a challenge method when performing DCR", async () => {
+    const options = getMockOptions();
+    const myFetch = getSuccessfulFetch() as jest.Mock<any, any>;
```

Comment:
The `any` here trigger ESLint warnings - might want to apply a more accurate type if possible, or add an ESLint exception if not.